core/lib/src/catcher/catcher.rs

```diff
@@ -354,6 +393,7 @@ impl fmt::Debug for Catcher {
             .field("base", &self.base)
             .field("code", &self.code)
             .field("rank", &self.rank)
+            .field("type_id", &self.type_id.as_ref().map(|_| "TY"))
```

Comment:
Why not the following?

```suggestion
            .field("type_id", &self.type_id)
```

Reply:
The `TypeId` doesn't actually provide much (if any) useful information, since it looks something like `TypeId(0x474ccf3b5db264ef53916706f7d7bb2c)`. I've updated this to actually print the type name (and, since I have the actual type name, it's now printed as part of the catcher on launch).